source/_integrations/apsystems.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: APsystems EZ1
```

Comment:
In the code, this integration is called [APSystems](https://github.com/home-assistant/core/pull/114531/files#diff-b5046a6076f032f4a62a5fe1c6bd17f5282a2249e99ba007ec4de67957aef296R3), not APSystems EZ1. The names should be identical.